dashboard-app/bower.json

```diff
@@ -33,6 +33,7 @@
     "angular-xeditable": "~0.5.0",
     "ng-js-tree": "~0.0.7",
     "angular-ui-select": "^0.19.6",
+    "angular-moment-picker": "~0.9.10"
```

Comment:
Even with comma I am not able to do bower install. When I save this component using following command:
bower install moment-picker --save
i get this in bower.json file:
    "angular-moment-picker": "moment-picker#^0.9.10"
so this is probably correct setting, but check it out